action/entrypoint.sh

```diff
@@ -2,6 +2,8 @@
 
 # Enable the matcher.
 ACTION_FOLDER=$(dirname ${0})
+sed -i "s/{{error_classes}}/${INPUT_ERROR_CLASSES//,/}/g" "${ACTION_FOLDER}/flake8-matcher.json"
```

Comment:
I am never a really big fan of those `${//,/}` things, as they tend to only work on `bash` (and not many people use `bash` these days). No clue if that is the case here .. I just gave up on it long ago :P Maybe just `sed` it?

Reply:
Ups i just changed it to use bash :P (for some reason Github is not sending me notifications for this things)

I actually started with `sed` and then decided to changed it to bash. Btw, it's my first time hearing that not many people use it :P

Reply:
I'll go back to sh and sed, no worries